wanda/build_input_test.go

```diff
@@ -0,0 +1,61 @@
+package wanda
+
+import (
+	"testing"
+
+	"reflect"
+)
+
+func TestBuildInputTags(t *testing.T) {
+	ts := newTarStream()
+
+	in := newBuildInput(ts, nil)
+
+	in.addTag("myimage")
+	in.addTag("cr.ray.io/rayproject/ray")
+	in.addTag("myimage") // duplicate
+
+	tagList := in.tagList()
+	want := []string{
+		"cr.ray.io/rayproject/ray",
+		"myimage",
+	}
+	if !reflect.DeepEqual(tagList, tagList) {
+		t.Errorf("got %v, want %v", tagList, want)
+	}
+}
+
+func TestBuildInputCore(t *testing.T) {
+	ts := newTarStream()
+	ts.addFile("Dockerfile", nil, "testdata/Dockerfile")
+
+	in := newBuildInput(ts, []string{"MESSAGE=test-msg"})
```

Comment:
can we try a map that has two '=' character, e.g. MESSAGE=test-msg=something

Reply:
done.

Reply:
change to MESSAGE=test=msg